packages/backend-common/package.json

```diff
@@ -70,7 +70,7 @@
     "@types/archiver": "^3.1.1",
     "@types/compression": "^1.7.0",
     "@types/concat-stream": "^1.6.0",
-    "@types/fs-extra": "^9.0.3",
+    "@types/fs-extra": "9.0.1",
```

Comment:
@Rugvip, is this change going to cause an issue?

Unfortunately, in order for me to get `tsc` to work, I needed to revert this and hard pin it back to `9.0.1`. See: f5908b983a0e8ff9e3337fda267a20474fb31baa for more details.